script/oci-runtime-validation

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+set -x
+
+REPOPATH="$( cd "$(dirname "$0")/.." ; pwd -P )"
```

Comment:
This seems overly complicated.  The `Makefile` calling this script knows where the compiled `runc` is.  Can't we set the `RUNTIME` enviroment variable in the `Makefile` rule?

```
RUNTIME=... script/oci-runtime-validation
```

Reply:
Good idea, I'll try that.